tools/sdg/topics/core_topics.py

```diff
@@ -402,6 +424,21 @@ def main(_):
     common.write_topic_json(_UNDATA_TOPIC_JSON, nodes)
     write_nl_descriptions(_UNDATA_NL_DESCRIPTIONS_FILE, nodes,
                           _UNDATA_SERIES_TOPICS_FILE, vars)
+  elif FLAGS.dc == 'undata_ilo':
+    load_variables(vars_file=_UNDATA_ILO_VARIABLES_FILE,
+                   vars=vars,
+                   var_prefix='',
+                   topic_prefix='dc/topic/',
+                   svpg_prefix='dc/svpg/ILO')
+    write_non_country_vars(_UNDATA_ILO_NON_COUNTRY_VARS, vars)
+    nodes = generate([_UNDATA_ILO_ROOT],
+                     vars,
+                     REMOVE_SVG_PREFIX,
+                     skip_assert=True)
```

Comment:
Add a comment for why we skip_assert?  I suppose we want to resolve that (in the next iteration of variable groupings?)